cartographer/mapping_3d/pose_graph/landmark_cost_function_test.cc

```diff
@@ -66,6 +66,7 @@ TEST(LandmarkCostFunctionTest, SmokeTest) {
                           jacobians_ptrs.data());
   EXPECT_THAT(residuals, ElementsAre(DoubleEq(1.), DoubleEq(0.), DoubleEq(0.),
                                      DoubleEq(0.), DoubleEq(0.), DoubleEq(0.)));
+  delete cost_function;
```

Comment:
Let's just make it an std::unique_ptr above.

Reply:
Done, PTAL